sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -253,6 +255,12 @@ export type ListParticipantsOptions = RestListParticipantsOptions;
 // @public
 export type ListReadReceiptsOptions = RestListReadReceiptsOptions;
 
+// @public
+export interface MessagingPolicy {
+    // (undocumented)
```

Comment:
Please add the description for the property.

Reply:
Updated with comment